package.json

```diff
@@ -96,27 +99,31 @@
     "eslint": "^6.8.0",
     "eslint-plugin-import": "^2.20.0",
     "fork-ts-checker-webpack-plugin": "^3.1.1",
+    "husky": "^4.2.5",
     "jest": "^25.3.0",
     "jest-cli": "^25.3.0",
     "jsdom": "16.2.2",
     "jsdom-global": "3.0.2",
+    "lint-staged": ">=10",
     "mocha": "^7.1.1",
     "node-fetch": "^2.6.0",
     "node-loader": "^0.6.0",
+    "node-sass": "^4.14.1",
+    "prettier": "^2.0.5",
     "react-test-renderer": "^16.13.1",
+    "sass-loader": "^9.0.1",
     "spectron": "^10.0.1",
     "style-loader": "^0.23.1",
     "ts-jest": "^25.4.0",
-    "ts-loader": "^6.2.1",
+    "ts-loader": "^6.2.2",
     "ts-node": "^8.10.1",
-    "typescript": "^3.7.0",
```

Comment:
were these changes introduced by eslint? if this is the case, I'd like to have them in different commit for better code review